action/attach_disk.go

```diff
@@ -7,19 +7,21 @@ import (
 	bslcvm "github.com/cloudfoundry/bosh-softlayer-cpi/softlayer/vm"
 )
 
-type AttachDisk struct {
+type AttachDiskAction struct {
 	vmFinder   bslcvm.Finder
 	diskFinder bslcdisk.Finder
 }
 
-func NewAttachDisk(vmFinder bslcvm.Finder, diskFinder bslcdisk.Finder) AttachDisk {
-	return AttachDisk{
-		vmFinder:   vmFinder,
-		diskFinder: diskFinder,
-	}
+func NewAttachDisk(
```

Comment:
If you are going to change the name to include `Action` then why not also rename the `New...` function to also include the `Action` suffix? Consistency is important. Also, are all the other also named with `Action`?